content/250-postgres/1200-more/1000-faq.mdx

```diff
@@ -73,6 +75,143 @@ Yes, read and write queries are counted equally as _operations_ and are billed t
+  })
+
+  // Array transaction: counted as 1 operation
+  await prisma.$transaction([
+    prisma.user.create({ data: { name: 'Alice' } }),
+    prisma.post.create({ data: { title: 'Hello', authorId: 1 } }),
+  ])
+  ```
+
+If a later query needs the result of an earlier one (for example, you need the user ID you just created), stick with an interactive transaction for correctness. Otherwise, batching and array transactions let you collapse multiple queries into a single billed operation, keeping both your operation count, and your cost down.
+
+### Is there a sample workload to estimate my expected charges?
+
+We will demonstrate three example workloads and estimate bills for small, medium, and large workloads. Each combine a realistic number of monthly active users (MAUs), a typical level of daily activity per user, and a rounded storage footprint. Here are the assumptions about the workloads:
+
+- **Small**: A hobby or early-stage side-project with ~`500` MAUs. Each user clicks around `10` times a day, and the entire database uses about `0.5` GB of storage.
+- **Medium**: A growing SaaS product serving ~`5000` MAUs. Power users average `40` actions per day, and the app stores roughly `6` GB of data.
+- **Large**: A production-grade, consumer-facing application handling ~`50000` MAUs. Heavy usage with `60` actions per user per day drives significant traffic, and the dataset reaches ~`40` GB.
+
+
+:::note[Plan rates]
+
+Here are the rates for each pricing plan:
+
+- **Starter plan** - \$18 per million operations
+  - Storage - 1 GB free then \$2 per additional GB
+- **Pro plan** - \$8 per million operations
+  - Storage - 5 GB free then \$1.5 per additional GB
+- **Business Annual plan** - \$4 per million operations
+  - Storage - 10 GB free then \$1 per additional GB
+
+You can also learn more about the rates for each plan on the [pricing page](https://www.prisma.io/pricing?utm_source=docs).
+
+:::
+
+We will use the following equations to estimate the monthly bill:
+
+```
+total_ops            = MAUs x actions_per_user_per_day x 30      
+billable_ops         = MAX(total_ops - 100_000, 0)
+ops_cost             = (billable_ops ÷ 1_000_000) x plan_rate
+
+billable_storage_GB  = MAX(storage_used_GB - free_storage_for_plan, 0)
+storage_cost         = billable_storage_GB x storage_rate_for_plan
+
+total_monthly_cost   = ops_cost + storage_cost + base_plan_fee
+```
+
+:::note
+
+You can place in your own MAU count, activity level, and storage footprint to project costs on any plan using the equations above.
+
+:::
+
+We will associate each workload with a plan and its corresponding rates, for example, the **Starter plan** for the small workload, the **Pro plan** for the medium workload, and the **Business Annual plan** for the large workload. Then we will apply the equations to the example workloads to generate a rough estimate of a monthly bill. For example:
+
+Example of a small workload on the Starter plan:
```

Comment:
Can we make these lines bold to better visualize the section of what the calculations are for different workloads?